ui/page/settings/app_settings_page.go

```diff
@@ -647,6 +680,83 @@ func (pg *AppSettingsPage) HandleUserInteractions() {
 			pg.ParentWindow().ShowModal(currentPasswordModal)
 		}
 	}
+
+	if pg.backupDEX.Clicked() {
+		// Show modal asking for dex password and then reveal the seeds.
+		dexPasswordModal := modal.NewCreatePasswordModal(pg.Load).
+			EnableName(false).
+			EnableConfirmPassword(false).
+			Title(values.String(values.StrDexPassword)).
+			SetPositiveButtonCallback(func(_, password string, pm *modal.CreatePasswordModal) bool {
+				var err error
+				defer func() {
+					if err != nil {
+						pm.SetError(err.Error())
+						pm.SetLoading(false)
+					}
+				}()
+
+				dexc := pg.AssetsManager.DexClient()
+				err = dexc.Login([]byte(password))
+				if err != nil {
+					return false
+				}
+
+				seed, err := dexc.ExportSeed([]byte(password))
+				if err != nil {
+					return false
+				}
+
+				pg.showDEXSeedModal(seed)
+				return err == nil
+			})
+
+		dexPasswordModal.SetPasswordTitleVisibility(false)
+		pg.ParentWindow().ShowModal(dexPasswordModal)
+	}
+}
+
+func (pg *AppSettingsPage) handleCopyEvent(gtx C) {
+	if pg.copyDEXSeed.Clicked() {
+		clipboard.WriteOp{Text: pg.dexSeed}.Add(gtx.Ops)
+		pg.copyDEXSeed.Text = values.String(values.StrCopied)
+		pg.copyDEXSeed.Color = pg.Theme.Color.Success
+		time.AfterFunc(time.Second*3, func() {
+			pg.copyDEXSeed.Text = values.String(values.StrCopy)
+			pg.copyDEXSeed.Color = pg.Theme.Color.Primary
+			pg.ParentWindow().Reload()
+		})
+	}
+}
+
+func (pg *AppSettingsPage) showDEXSeedModal(seed dex.Bytes) {
+	defer utils.ZeroBytes(seed)
+	pg.dexSeed = stringifySeed(seed)
```

Comment:
pg.dexSeed should be []byte, not string. So you can zero once the modal is closed.